src/main/java/gameframework/assets/Sound.java

```diff
@@ -0,0 +1,83 @@
+package gameframework.assets;
+
+import java.io.BufferedInputStream;
+import java.io.IOException;
+
+import javax.sound.sampled.AudioFormat;
+import javax.sound.sampled.AudioInputStream;
+import javax.sound.sampled.AudioSystem;
+import javax.sound.sampled.Clip;
+import javax.sound.sampled.DataLine;
+import javax.sound.sampled.LineUnavailableException;
+import javax.sound.sampled.UnsupportedAudioFileException;
+
+/**
+ * Class allowing to create sounds and play them. It is not intended to play musics.
+ */
+public class Sound {
+
+	/**
+	 * The sound clip instance
```

Comment:
please quickly summarize what is a clip and what you are going to use it for. Your current comment is quite useless